presto-base-jdbc/pom.xml

```diff
@@ -143,6 +143,14 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-main</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+            <type>test-jar</type>
```

Comment:
this line is no necessary (I guess)

Reply:
It is probably needed since I need to access TestingConnectorSession which lies in the test directory of presto-main and not in the project source 🤔But there might be another way to access it than the `<type>test-jar</type>` property.